pyproject.toml

```diff
@@ -118,8 +103,17 @@ core = [
 ]
 
 check = [
+	# upstream
 	"pytest-checkdocs >= 2.4",
 	"pytest-ruff >= 0.2.1; sys_platform != 'cygwin'",
+
+	# local
+
+	# require newer pytest-ruff than upstream for pypa/setuptools#4368
+	# also exclude cygwin for pypa/setuptools#3921
+	'pytest-ruff >= 0.3.2; sys_platform != "cygwin"',
```

Comment:
This was actually for https://github.com/businho/pytest-ruff/issues/17 , which was re-opened because the output wasn't actually cleaned up. So this extra requirement is unneeded.

Should I remove it now ?
```suggestion
```

Otherwise I've also done it in https://github.com/pypa/setuptools/pull/4562

Reply:
Sounds like a fair assessment. Thank you very much @Avasam.

We can do it in this PR on in any other following one.